lib/vrl/stdlib/Cargo.toml

```diff
@@ -33,6 +33,7 @@ once_cell = { version = "1.10", optional = true }
 rand = { version = "0.8.5", optional = true }
 regex = { version = "1", optional = true }
 rust_decimal = { version = "1", optional = true }
+serde = { version = "1", default-features = false, features = ["derive"] }
```

Comment:
Could you make this `optional = true`, and then add it to the list of required dependencies for the `is_json` feature?